shared/src/main/scala/za/co/absa/hyperdrive/shared/utils/FileUtils.scala

```diff
@@ -33,6 +35,15 @@ private[hyperdrive] object FileUtils {
 
   def isEmpty(directory: String)(implicit fs: FileSystem): Boolean = {
     val path = new Path(directory)
-    fs.exists(path) &&  !fs.listFiles(path, true).hasNext
+    fs.exists(path) && !fs.listFiles(path, true).hasNext
+  }
+
+  def readFileLines(file: String): Vector[String] = {
```

Comment:
This method is only used in tests. Please move it to a class in the test folder

Reply:
Moved